### DISCUSSION TITLE
add icons to sorting types in song select

### DISCUSSION THREAD

**Discussion by Woshuojidan** [ppy/osu#30996] [Ideas]
![image](https://github.com/user-attachments/assets/fc0724cc-6885-4e3c-89da-3bac0ea21492)
there's a solid amount of sorting types, i think there should be some type of icon representing each sorting type on the left of each to make it easier to read and pick out the one you want. (maybe this could help people with reading disabilities)

**Comment by cl8n** [comment]
long unorganized lists like this are hard to scan regardless of iconography imo. will experiment with that but I think it may be more effective to try to find some grouping for these options rather than a single list
